lib/grape-swagger/endpoint.rb

```diff
@@ -287,17 +287,13 @@ def expose_params_from_model(model)
       model_name
     end
 
-    def model_name(name)
-      if name.respond_to?(:entity_name)
-        name.entity_name
-      elsif name.to_s.end_with?('Entity', 'Entities')
-        length = 0
-        name.to_s.split('::')[0..-2].reverse.take_while do |x|
-          length += x.length
-          length < 42
-        end.reverse.join
+    def model_name(entity)
```

Comment:
This will still make assumptions about the grape / entity / class structure, can we just leave it as entity.name or entity.entity_name if defined?
There is no reference in the spec that definitions should have a specific naming convention.

I argue for this point that this will be the most transparent to the developer. For internal documentation it helps to correctly show the entity name (fully).

For example take: `AnimalFarm::Internal::CowEntity` and `AnimalFarm::Internal:GoatEntity` both would be rendered as `AnimalFarm::Internal`